pkg/controller/controller.go

```diff
@@ -352,6 +359,9 @@ func (c *controller) updateBrokerReadyCondition(broker *v1alpha1.Broker, status
 	} else {
 		glog.V(5).Infof("Updated ready condition for Broker %v to %v", broker.Name, status)
 	}
+	// TODO: all the information is available at this level except the proper event type level, I propose
+	// we simply pass that in.
+	c.recorder.Eventf(broker, api.EventTypeNormal, reason, "Broker condition set to %v (%v)", status, message)
```

Comment:
@pmorie what do you think about doing it this way rather than making an event that essentially duplicates the parameters of this function?